docker/build.sh

```diff
@@ -8,54 +8,16 @@ SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
 # Cd into $FF_HOME. Assumes this script is in $FF_HOME/docker
 cd "$SCRIPT_DIR/.."
 
-cuda_version="empty"
-image="flexflow"
+cuda_version=${cuda_version:-11.1.1}
```

Comment:
can we make the `cuda_version` environment variable take CUDA versions with only one decimal, like the previous cuda_version argument? then, we can add the default subversion digit in the script